staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -307,6 +307,10 @@ func (s *GenericAPIServer) installAPIResources(apiPrefix string, apiGroupInfo *A
 		}
 
 		apiGroupVersion := s.getAPIGroupVersion(apiGroupInfo, groupVersion, apiPrefix)
+		if len(apiGroupVersion.Storage) == 0 {
+			continue
```

Comment:
`apiGroupInfo.GroupMeta.GroupVersions` always has all versions, even if some are disabled. Compare the NewREST code.